instructors.md

```diff
@@ -7,196 +7,245 @@ subtitle: Instructor's Guide
+    a very clear illustration to the students as to who does what.
+
+*   It is also effective to pair up students during this lesson and assign one
+    member of the pair to take the role of the owner and the other the role of
+    the collaborator.  In this setup, challenges can include asking the
+    collaborator to make a change, commit it, and push the change to the remote
+    repository so that the owner can then retrieve it, and vice-versa.  The
+    role playing between the instructors can get a bit "dramatic" in the
+    conflicts part of the lesson if the instructors want to inject some humor
+    into the room.
+
+*   If you don't have two projectors, have two instructors at the front of the
+    room.  Each instructor does their piece of the collaboration demonstration
+    on their own computer and then passes the projector cord back and forth
+    with the other instructor when it's time for them to do the other part of
+    the collaborative workflow.  It takes less than 10 seconds for each
+    switchover, so it doesn't interrupt the flow of the lesson.
+    And of course it helps to give each of the instructors a different-colored
+    hat, or put different-colored sticky notes on their foreheads.
+
+*   If you're the only instructor, the best way to create is clone the two
+    repos in your Desktop, but under different names, e.g., pretend one is your
+    computer at work: 
+
+    ~~~{.input}    
+    $ git clone https://github.com/vlad/planets.git planets-at-work
+    ~~~
+
+*   It's very common that learners mistype the remote alias or the remote URL
+    when adding a remote, so they cannot `push`. You can diagnose this with
+    `git remote -v` and checking carefully for typos. 
+    - To fix a wrong alias, you can do `git remote rename <old> <new>`. 
+    - To fix a wrong URL, you can do `git remote set-url <alias> <newurl> `.
+
+*   Before cloning the repo, be sure that nobody is inside another repo. The
+    best way to achieve this is moving to the `Desktop` before cloning: `cd &&
+    cd Desktop`.
+
+*   If both repos are in the `Desktop`, have them to clone their collaborator
+    repo under a given directory using a second argument: 
+
+    ~~~{.input}    
+    $ git clone https://github.com/vlad/planets.git vlad-planet`
+    ~~~
+    
+*   The most common mistake is that learners `push` before `pull`ing. If they
+    `pull` afterward, they may get a conflict.
+
+*   Conflicts, sometimes weird, will start to arise. Stay tight: conflicts are
+    next.
 
 ## [Conflicts](09-conflict.html)
 
-FIXME
+*   Expect the learners to make mistakes. Expect *yourself* to make mistakes. This
+    happens because it is late in the lesson and everyone is tired.
 
-## [Open Science](10-open.html)
+*   If you're the only instructor, the best way to create a conflict is:
 
-FIXME
+    - Clone your repo in a different directory, pretending is your computer at
```

Comment:
Yes, I agree. But I put this line, because it's actually hard to create a conflict if you are the only instructor. I think it's just is late in the lesson, so I wanted to have a reminder on how to do it.